templates/layout.html

```diff
@@ -35,7 +35,7 @@ <h1 id="logo"><a id="tb-opened" class="button special" style="margin-left: -15px
                 for href, title, print in [
                   ['/blog', 'Blog', 1],
                   ['/projekte', 'Projekte', 1],
-                  ['/verein', 'Verein', 1],
+                  ['/vereinsheim', 'Verein', 1],
```

Comment:
Finde ich gut dass man da auf die Vereinsheim Seite kommt, aber ich glaube dann sollte man die "verein" seite umbenennen in "mitgliedschaft" oder so. Man kommt ja auch nur noch über den Button "Satzung und Ordnung" auf die /verein Seite.

Reply:
+1